app/views/users/show.html.erb

```diff
@@ -32,7 +38,7 @@
   <%= @user.bio %>
 </p>
 
-<%= link_to 'Edit', edit_user_path(@user) %> |
-<%= link_to 'Back', users_path %>
+<%= link_to 'Edit', edit_user_path(@user.id) %> | 
+<%= link_to 'Add A Vehicle', new_user_vehicle_path(@user.id) %>
 
 </div>
```

Comment:
What would you think about adding links to the choose ride page from the profile? I don't know if it's a good idea or not - just trying to brainstorm some user experience stuff.

Reply:
All agreed that 'Book A Ride' will go on profile page and hamburger menu

Reply:
Good thinking! I think we should probably have a link to set up a ride both on the profile page as well as in the hamburger menu considering that is our apps primary purpose.